### PR TITLE
Update report_tags.md

### PR DESCRIPTION
**Pull request by xxlongzaixx**
add description

### CODE REVIEW

File: report_tags.md [hunk 1]
```diff
@@ -2,222 +2,222 @@
...
+| Cross Chain | Cross Chain refers to interactions or transactions that occur between different blockchain networks. This introduces a unique set of security considerations, as smart contracts must implement measures to validate and handle cross-chain interactions securely. |
+| Cross Chain Message | Cross Chain Message refers to a communication or transaction initiated on one blockchain network that is intended to interact with or trigger actions on a different blockchain. |
 | Cryptography | Cryptography vulnerabilities occur when a smart contract employs flawed or insufficient cryptographic practices. In the context of blockchain and smart contracts, this often involves the use of weak or inappropriate algorithms for tasks such as generating random numbers. The use of improper cryptography can compromise the security and integrity of a smart contract.|
-| CryptoPunks | |
+| CryptoPunks | CryptoPunks is a non-fungible token collection on the Ethereum blockchain，launched as a fixed set of 10,000 items in mid-2017 and became one of the inspirations for the ERC-721 standard. |
```
[the remark this answers is not in the document]
Comment (hans-cyfrin) [reply report_tags.md:57]: No worries, we will merge and update on our side.
We really appreciate your contribution!

**Review by hans-cyfrin — APPROVED** [review]

**Review by wang1252 — APPROVED** [review]

**Comment by wang1252** [comment]
Great work.
Thank you @xxlongzaixx

**Comment by xxlongzaixx** [comment]
Can I get an Updraft invite code, please. I got nobody to invite

**Comment by hans-cyfrin** [comment]
DM me on Twitter

**Pull request merged**